main.py

```diff
@@ -76,18 +101,18 @@
 
 # NC does not have a 'predict_proba' attribute (i.e. no probability estimates)
 # Probability estimates are not available for hinge loss in SGD. See learner_functions for a modified SGD trainer function.
-sgd_gender_mod, sgd_gender_score_mod = lf.train_sgd_mod(data.proteomic,gender_labels)
-sgd_msi_mod, sgd_msi_score_mod = lf.train_sgd_mod(data.proteomic,MSI_labels)
-
-
-print("\n\n******************************************************************")
-print("SOFT VOTE FOR GENDERS")
-gender_estimators = [('knn', knn_gender), ('lr', lr_gender), ('rf', rf_gender), ('sgd', sgd_gender_mod), ('mlp', mlp_gender)]
-sv_gender = sv.soft_vote(gender_estimators, data.proteomic, gender_labels)
-print("******************************************************************")
-
-print("\n\n******************************************************************")
-print("SOFT VOTE FOR MSI")
-msi_estimators = [('knn', knn_msi), ('lr', lr_msi), ('rf', rf_msi), ('sgd', sgd_msi_mod), ('mlp', mlp_msi)]
-sv_msi = sv.soft_vote(msi_estimators, data.proteomic, MSI_labels)
-print("******************************************************************")
+# sgd_gender_mod, sgd_gender_score_mod = lf.train_sgd_mod(protein_sub_set,gender_labels)
```

Comment:
We're probably OK to just delete this instead of commenting it out since it's tracked in Git.